src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/DeleteInstanceV36.sql

```diff
@@ -0,0 +1,266 @@
+﻿/***************************************************************************************/
+-- STORED PROCEDURE
+--     DeleteInstanceV36
+--
+-- FIRST SCHEMA VERSION
+--     36
+--
+-- DESCRIPTION
+--     Removes the specified instance(s) and places them in the DeletedInstance table for later removal
+--
+-- PARAMETERS
+--     @partitionKey
+--         * The Partition key
+--     @cleanupAfter
+--         * The date time offset that the instance can be cleaned up.
+--     @createdStatus
+--         * Status value representing the created state.
+--     @studyInstanceUid
+--         * The study instance UID.
+--     @seriesInstanceUid
+--         * The series instance UID.
+--     @sopInstanceUid
+--         * The SOP instance UID.
+/***************************************************************************************/
+CREATE OR ALTER PROCEDURE dbo.DeleteInstanceV36
```

Comment:
We don't need tp version if the params are not changing

Reply:
Done